homeassistant/components/tomorrowio/sensor.py

```diff
@@ -289,6 +292,32 @@ async def async_setup_entry(
     async_add_entities(entities)
 
 
+def smart_round(value: float) -> float:
+    """Round a float to have at least two digits."""
```

Comment:
Is this function intended to guess the precision of a sensor based on the magnitude of its value?
I really don't think that's correct, the precision should be a property of the sensor, compensated for by unit conversion.

Reply:
Here's an example of how to do the compensation: https://github.com/home-assistant/core/blob/c2c6d12d7300169ce156d63cb33e3be299315cd0/homeassistant/components/sensor/__init__.py#L430-L439

Reply:
I've decided to remove this and just keep it simple

Reply:
Is there some unit conversion happening where the units are orders of magnitude different? If there is, it makes sense to not always round to 2 digits.

Reply:
ppb -> ppm but do we really need that level of precision? In any event that can be a feature for a separate PR, it shouldn't have been lumped into this one in retrospect